src/core/2d_primitives.js

```diff
@@ -390,12 +390,16 @@ p5.prototype.quad = function() {
  * @param  {Integer} [detailY] number of segments in the y-direction
  * @chainable
  */
-p5.prototype.rect = function(x, y, w, h, detailX, detailY) {
+p5.prototype.rect = function(x, y, w, h) {
   p5._validateParameters('rect', arguments);
 
   if (this._renderer._doStroke || this._renderer._doFill) {
     var vals = canvas.modeAdjust(x, y, w, h, this._renderer._rectMode);
-    this._renderer.rect([vals.x, vals.y, vals.w, vals.h, detailX, detailY]);
+    var args = [vals.x, vals.y, vals.w, vals.h];
+    for (var i = 4; i < arguments.length; i++) {
```

Comment:
can you add a comment here about what's happening, so it's clear that the additional args are being added on to the end, unmodified?